src/build.ts

```diff
@@ -484,7 +484,7 @@ function startRollupWatcher(nitro: Nitro, rollupConfig: RollupConfig) {
         nitro.hooks.callHook("compiled", nitro);
         const { framework } = nitro.options;
         nitro.logger.success(
-          `${framework.name} v${framework.version} built`,
+          `Nitro Server v${nitroVersion} built for ${framework.name} v${framework.version}`,
```

Comment:
This is good! We only now to check if `framework.name !== "nitro"` to avoid showing `Nitro server v1 build for Nitro v1` :D